cloud-config/aws/README.md

```diff
@@ -0,0 +1,156 @@
+This Cloud-Config setup is intended to be used with the Ubuntu Server 18.04 LTS AMI in AWS. You can search for the latest AMI id for your AWS zone at https://cloud-images.ubuntu.com/locator/ec2. Search for `18.04 amd64 hvm:ebs-ssd`
+
+**To use this setup, you must able to link a domain name to EC2 machine's IP address. If you don't own a domain, services such as [FreeDNS](https://freedns.afraid.org) offer free sub-domains under a range of domains.**
+
+## Prerequisites
+
+**Warning**: Make sure at all time that you have selected the availability zone where you want to perform your actions. You can choose the availability zone using the dropdown menu at the top-right corner of the AWS console website.
+
+### Create a VPC
```

Comment:
I would expect most people would use the default VPC. If this is best practice, or makes other things harder, maybe it's not too bad.

Reply:
I believe you can't do that anymore... I'll check

Reply:
Yeah, unless theres some hidden way I don't know about, I've confirmed you can't complete the creation of the EC2 machine without providing a VPC.